src/main/java/raisetech/StudentManagement/controller/StudentController.java

```diff
@@ -34,20 +37,14 @@ public StudentController(StudentService service, StudentConverter converter) {
   }
 
   @GetMapping("/studentList")
-  public String getStudentList(Model model) {
+  public List<StudentDetail> getStudentList() {
     List<Student> students = service.searchStudentList();
     List<StudentsCourses> studentsCourses = service.searchStudentsCoursesList();
-
-    model.addAttribute("studentList", converter.convertStudentDetails(students, studentsCourses));
-    return "studentList";
-  }
-
-  private List<StudentsCourses> getStudentsCourses() {
-    return service.searchStudentsCoursesList();
+    return converter.convertStudentDetails(students, studentsCourses);
   }
 
-  @GetMapping("/studentInformation/{id}")
-  public String getStudentInformation(@PathVariable long id, Model model) {
+  @GetMapping("/student/{id}")
```

Comment:
RESTでつくるなら名詞の複数形がいいですね。
```suggestion
  @GetMapping("/students/{id}")
```

Reply:
他のAPIも REST で作るならパスは見直したい。

Reply:
パスの見直しは別の機会に対応しましょうか。

Reply:
分かりました。私なりにパスの見直しについて勉強しときます。